scripts/gcp-build-and-push.sh

```diff
@@ -38,6 +61,28 @@ for tag in "$version-commercial" "beta" "$timestamp"; do
     docker push "$image:$tag"
 done
 
+# Install gcrane if not already installed (only if we have regions to replicate to)
+if [ ${#REPLICA_REGIONS[@]} -gt 0 ]; then
+    if ! command -v gcrane &> /dev/null; then
+        echo "Installing gcrane..."
+        go install github.com/google/go-containerregistry/cmd/gcrane@latest
+        export PATH="$(go env GOPATH)/bin:$PATH"
+    fi
+    
+    # Replicate images to all configured regions
+    for region in "${REPLICA_REGIONS[@]}"; do
+        echo "Replicating images to ${region}..."
+        replica_image="${region}-docker.pkg.dev/lightdash-containers/lightdash/lightdash"
+        
+        for tag in "$version-commercial" "beta" "$timestamp"; do
+            echo "  Replicating tag ${tag} to ${region}..."
+            gcrane cp "$image:$tag" "$replica_image:$tag"
```

Comment:
another option could be to "push" directly to another region, that could even save traffic between regions. 

perhaps by adding `-t "$replica_image:$version-commercial" ` on line #35